install/kubernetes/cilium/templates/hubble/tls-certmanager/metrics-server-secret.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.hubble.enabled .Values.hubble.metrics.tls.enabled .Values.hubble.tls.auto.enabled (eq .Values.hubble.tls.auto.method "certmanager") }}
-{{- $cn := list (.Values.cluster.name | replace "." "-") "hubble-metrics.cilium.io" | join "." }}
+{{- $cn := default "*.hubble-metrics.cilium.io" .Values.hubble.metrics.tls.server.commonName }}
```

Comment:
Apologies that I missed that before. This one still looks like it changes the common name of existing certificates - the old one contains the cluster name, while the new one doesn't. Given that this looks intentional it is probably best to preserve the old behavior.